### DISCUSSION TITLE
为什么会报这个错？

### DISCUSSION THREAD

**Discussion by ljmeva** [the1812/Bilibili-Evolved#3076] [提问 / Q&A] [answered]
### 对应功能
安装过程
### 问题描述
在安装完脚本后，脚本运行报错，同样的步骤，我在其他电脑上可以正常使用，在我的电脑上就出问题了，我换了火狐浏览器也不行，也是同样的报错
### 脚本版本
v2.1.5
### 浏览器版本
edge版本 98.0.1108.62 (官方内部版本) (64 位)
### 错误信息
Uncaught (in promise) ReferenceError: _ is not defined
### 附加截图
![image](https://user-images.githubusercontent.com/96721423/156355327-d9c56981-e3c0-48a6-906f-c1e65a9ed42a.png)
![image](https://user-images.githubusercontent.com/96721423/156355475-f35ce330-f979-4f10-b31d-d77365a39381.png)

**Comment by ljmeva** [comment]
![image](https://user-images.githubusercontent.com/96721423/156356194-b80ecb8a-8dbb-4e14-9114-fda43488f005.png)
忘了还有这张图，b站左侧没有齿轮，我重新安装过很多次，依旧是同样的

**Comment by the1812** [comment, marked as answer]
#3020

**Comment by ljmeva** [comment]
解决了，感谢作者大大
